docs/machines/aurora/data-management/moving_data_to_aurora/globus.md

```diff
@@ -0,0 +1,35 @@
+# Transfering files through Globus
+
+## During Acceptance Testing period
+
+We have set up a temporary Globus endpoint for Flare that you can use to transfer data out. The endpoint is called "alcf#dtn_flare_at" and is set up for read-only access. The endpoint will be available through the acceptance testing (AT) period and will be shutdown after AT concludes.
+
+## Before Acceptance Testing
+
+Currently, only Globus personal is supported on Aurora.
+Perform the following steps to transfer data to/from the Aurora login nodes.
+
+1. On a fresh connection to the login nodes, ensuring no proxies are being set
+   (which may require commenting the proxy settings in the `~/.bashrc` or
+   `~/.bash_profile` files), execute
+
+    ```bash
+    /soft/tools/proxychains/bin/proxychains4 -f /soft/tools/proxychains/etc/proxychains.conf /soft/tools/globusconnect/globusconnect -setup --no-gui
+    ```
```

Comment:
@felker so I just added the spacing so that its aligned under the 1.

e.g.

1. Item one
```bash
hostname
```

became

1. Item one

   ```bash
   hostname
   ```

   (\+ always blank line surrounding codeblocks)